edd-activecampaign.php

```diff
@@ -784,6 +792,49 @@ public function completed_download_purchase_signup( $download_id = 0, $payment_i
 			$payment->delete_meta( 'eddactivecampaign_activecampaign_signup' );
 		}
 	}
+		/**
+		 * Enqueues the JavaScript files.
+		 *
+		 * @since 1.1.2
+		 * @return void
+		 */
+		public function load_admin_scripts() {
+			wp_enqueue_script( 'load_admin_scripts', $this->plugin_url . 'assets/js/scripts.js', array( 'jquery' ) );
```

Comment:
```suggestion
			wp_enqueue_script( 'edd-activecampaign', $this->plugin_url . 'assets/js/scripts.js', array( 'jquery' ) );
```

That first parameter is like the ID of the script, so in this case it does need to be a prefixed, unique name for this JavaScript file.